dev-python/seaborn/seaborn-0.10.0.ebuild

```diff
@@ -15,7 +15,7 @@ SLOT="0"
 LICENSE="BSD"
 KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
 IUSE="test"
-RESTRICT="!test? ( test )"
+RESTRICT="test"
```

Comment:
please do this in a separate commit and not in the bump commit

Reply:
should I add the commit to this PR, or do another one after this is merged?

Reply:
well, ok, I just have the version bump in this commit now.